/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
Exception in thread Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>
    main()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 20, in main
    click.Group(commands=commands)(prog_name="bench")
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/commands/scheduler.py", line 203, in start_worker
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    start_worker(queue, quiet=quiet)Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner

  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/background_jobs.py", line 196, in start_worker
    Worker(queues, name=get_worker_name(queue)).work(logging_level=logging_level)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 626, in work
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
    self.register_death()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 326, in register_death
    p.execute()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 4019, in execute
    pubsub.get_message(ignore_subscribe_messages=True,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
    pubsub.get_message(ignore_subscribe_messages=True,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
    response = self.parse_response(block=False, timeout=timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
    response = self.parse_response(block=False, timeout=timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
    if not block and not conn.can_read(timeout=timeout):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read
        if not block and not conn.can_read(timeout=timeout):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read
return self._parser.can_read(timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
    return self.read_from_socket(timeout=timeout,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
        pubsub.get_message(ignore_subscribe_messages=True,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
    response = self.parse_response(block=False, timeout=timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
    if not block and not conn.can_read(timeout=timeout):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read
    return self._parser.can_read(timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
    return self.read_from_socket(timeout=timeout,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
        raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
return execute(conn, stack, raise_on_error)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3894, in _execute_transaction
    self.parse_response(connection, '_')    return self._parser.can_read(timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
    return self.read_from_socket(timeout=timeout,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3977, in parse_response
    result = Redis.parse_response(
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 915, in parse_response
    response = connection.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 739, in read_response
    response = self._parser.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 470, in read_response
    self.read_from_socket()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
Exception in thread Exception in thread Thread-1Thread-1Exception in thread :
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 427, in read_from_socket
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
:
Thread-1Traceback (most recent call last):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 427, in read_from_socket
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
:
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 427, in read_from_socket
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
    bufflen = recv_into(self._sock, self._buffer)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/_compat.py", line 75, in recv_into
        bufflen = recv_into(self._sock, self._buffer)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/_compat.py", line 75, in recv_into
bufflen = recv_into(self._sock, self._buffer)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/_compat.py", line 75, in recv_into
    return sock.recv_into(*args, **kwargs)
ConnectionResetError    pubsub.get_message(ignore_subscribe_messages=True,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
: [Errno 104] Connection reset by peer        
pubsub.get_message(ignore_subscribe_messages=True,
During handling of the above exception, another exception occurred:


Traceback (most recent call last):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return sock.recv_into(*args, **kwargs)
ConnectionResetError: [Errno 104] Connection reset by peerpubsub.get_message(ignore_subscribe_messages=True,

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
        return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
exec(code, run_globals)    exec(code, run_globals)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>

      File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>
response = self.parse_response(block=False, timeout=timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
    return sock.recv_into(*args, **kwargs)    response = self.parse_response(block=False, timeout=timeout)

ConnectionResetError  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
: [Errno 104] Connection reset by peer
    
During handling of the above exception, another exception occurred:

if not block and not conn.can_read(timeout=timeout):Traceback (most recent call last):

  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read
        return self._parser.can_read(timeout)return _run_code(code, main_globals, None,

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)    
return self.read_from_socket(timeout=timeout,  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
        main()main()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 20, in main
redis.exceptions.    click.Group(commands=commands)(prog_name="bench")
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 829, in __call__
ConnectionError: Connection closed by server.
    main()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 20, in main

  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/bench_helper.py", line 20, in main
    if not block and not conn.can_read(timeout=timeout):            
click.Group(commands=commands)(prog_name="bench")click.Group(commands=commands)(prog_name="bench")
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 829, in __call__
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 829, in __call__
response = self.parse_response(block=False, timeout=timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
    return self._parser.can_read(timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
    return self.read_from_socket(timeout=timeout,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
    if not block and not conn.can_read(timeout=timeout):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read
    return self._parser.can_read(timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
    return self.read_from_socket(timeout=timeout,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
    return self.main(*args, **kwargs)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 782, in main
    return self.main(*args, **kwargs)    
return self.main(*args, **kwargs)  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 782, in main

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
        rv = self.invoke(ctx)rv = self.invoke(ctx)

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
            return _process_result(sub_ctx.command.invoke(sub_ctx))return _process_result(sub_ctx.command.invoke(sub_ctx))return _process_result(sub_ctx.command.invoke(sub_ctx))


  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)        
return _process_result(sub_ctx.command.invoke(sub_ctx))return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/commands/scheduler.py", line 203, in start_worker

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
        return ctx.invoke(self.callback, **ctx.params)return ctx.invoke(self.callback, **ctx.params)

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 610, in invoke
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/click/core.py", line 610, in invoke
        return callback(*args, **kwargs)return callback(*args, **kwargs)

  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/commands/scheduler.py", line 203, in start_worker
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/commands/scheduler.py", line 203, in start_worker
    start_worker(queue, quiet=quiet)
          File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/background_jobs.py", line 196, in start_worker
start_worker(queue, quiet=quiet)start_worker(queue, quiet=quiet)

  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/background_jobs.py", line 196, in start_worker
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/background_jobs.py", line 196, in start_worker
    Worker(queues, name=get_worker_name(queue)).work(logging_level=logging_level)        
Worker(queues, name=get_worker_name(queue)).work(logging_level=logging_level)Worker(queues, name=get_worker_name(queue)).work(logging_level=logging_level)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 626, in work

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 626, in work
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 626, in work
    self.register_death()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 326, in register_death
        self.register_death()self.register_death()

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 326, in register_death
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 326, in register_death
    p.execute()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 4019, in execute
        p.execute()p.execute()

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 4019, in execute
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 4019, in execute
    return execute(conn, stack, raise_on_error)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3894, in _execute_transaction
    return execute(conn, stack, raise_on_error)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3894, in _execute_transaction
    return execute(conn, stack, raise_on_error)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3894, in _execute_transaction
    self.parse_response(connection, '_')
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3977, in parse_response
    result = Redis.parse_response(
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 915, in parse_response
        response = connection.read_response()self.parse_response(connection, '_')

  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 739, in read_response
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3977, in parse_response
    self.parse_response(connection, '_')
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3977, in parse_response
    response = self._parser.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 470, in read_response
    self.read_from_socket()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 452, in read_from_socket
    raise ConnectionError("Error while reading from socket: %s" %
redis.exceptions.ConnectionError: Error while reading from socket: (104, 'Connection reset by peer')
    result = Redis.parse_response(
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 915, in parse_response
    result = Redis.parse_response(
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 915, in parse_response
    response = connection.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 739, in read_response
    response = connection.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 739, in read_response
    response = self._parser.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 470, in read_response
    response = self._parser.read_response()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 470, in read_response
    self.read_from_socket()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 452, in read_from_socket
    self.read_from_socket()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 452, in read_from_socket
    raise ConnectionError("Error while reading from socket: %s" %
redis.exceptions.ConnectionError: Error while reading from socket: (104, 'Connection reset by peer')
    raise ConnectionError("Error while reading from socket: %s" %
redis.exceptions.ConnectionError: Error while reading from socket: (104, 'Connection reset by peer')
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
14:14:58 Traceback (most recent call last):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 1013, in perform_job
    rv = job.perform()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/job.py", line 709, in perform
    self._result = self._execute()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/job.py", line 732, in _execute
    result = self.func(*self.args, **self.kwargs)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/background_jobs.py", line 144, in execute_job
    method(**kwargs)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/core/doctype/user/user.py", line 1130, in create_contact
    contact.insert(
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/model/document.py", line 255, in insert
    self._validate_links()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/model/document.py", line 908, in _validate_links
    frappe.throw(_("Could not find {0}").format(msg), frappe.LinkValidationError)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/__init__.py", line 504, in throw
    msgprint(
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/__init__.py", line 479, in msgprint
    _raise_exception()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/__init__.py", line 434, in _raise_exception
    raise raise_exception(msg)
frappe.exceptions.LinkValidationError: Could not find User Id: mohan27895@gmail.com

14:15:00 Traceback (most recent call last):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/worker.py", line 1013, in perform_job
    rv = job.perform()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/job.py", line 709, in perform
    self._result = self._execute()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/rq/job.py", line 732, in _execute
    result = self.func(*self.args, **self.kwargs)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/utils/background_jobs.py", line 144, in execute_job
    method(**kwargs)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/core/doctype/user/user.py", line 1130, in create_contact
    contact.insert(
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/model/document.py", line 255, in insert
    self._validate_links()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/model/document.py", line 908, in _validate_links
    frappe.throw(_("Could not find {0}").format(msg), frappe.LinkValidationError)
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/__init__.py", line 504, in throw
    msgprint(
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/__init__.py", line 479, in msgprint
    _raise_exception()
  File "/home/eorchids/frappe-bench13/apps/frappe/frappe/__init__.py", line 434, in _raise_exception
    raise raise_exception(msg)
frappe.exceptions.LinkValidationError: Could not find User Id: mohan27895@gmail.com

/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/botocore/httpsession.py:18: DeprecationWarning: 'urllib3.contrib.pyopenssl' module is deprecated and will be removed in a future release of urllib3 2.x. Read more in this issue: https://github.com/urllib3/urllib3/issues/2680
  from urllib3.contrib.pyopenssl import orig_util_SSLContext as SSLContext
/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/botocore/httpsession.py:18: DeprecationWarning: 'urllib3.contrib.pyopenssl' module is deprecated and will be removed in a future release of urllib3 2.x. Read more in this issue: https://github.com/urllib3/urllib3/issues/2680
  from urllib3.contrib.pyopenssl import orig_util_SSLContext as SSLContext
/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/botocore/httpsession.py:18: DeprecationWarning: 'urllib3.contrib.pyopenssl' module is deprecated and will be removed in a future release of urllib3 2.x. Read more in this issue: https://github.com/urllib3/urllib3/issues/2680
  from urllib3.contrib.pyopenssl import orig_util_SSLContext as SSLContext
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3718, in run
    pubsub.get_message(ignore_subscribe_messages=True,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3617, in get_message
    response = self.parse_response(block=False, timeout=timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/client.py", line 3503, in parse_response
    if not block and not conn.can_read(timeout=timeout):
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 734, in can_read
    return self._parser.can_read(timeout)
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 416, in can_read
    return self.read_from_socket(timeout=timeout,
  File "/home/eorchids/frappe-bench13/env/lib/python3.10/site-packages/redis/connection.py", line 429, in read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
/home/eorchids/frappe-bench13/apps/frappe/frappe/build.py:7: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.spawn import find_executable
